mantle/kola/harness.go

```diff
@@ -969,6 +969,7 @@ func runTest(h *harness.H, t *register.Test, pltfrm string, flight platform.Flig
 		var userdata *conf.UserData = t.UserData
 
 		options := platform.MachineOptions{
+			MultiPathed:     t.MultiPathed,
```

Comment:
How about `MultiPathDisks` or so?  In a disk-specific context "multipath" is unambiguous, but here it's an option mixed in with things like RAM etc.

Reply:
I just kept it as `MultiPathDisk` so it matches the name it previously had. That way it's even more obvious that it's a straight "field promotion" from `QemuMachineOptions` to `MachineOptions`.